tests/tasks/setup/kitctl/setup-control-plane.yaml

```diff
@@ -0,0 +1,124 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: control-plane-setup
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/categories: Kubernetes
+    tekton.dev/tags: CLI, kubectl
+    tekton.dev/displayName: "kubernetes actions"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: |
+    Setup a kubernetes control plane in the guess cluster.
+  params:
+  - name: name
+    default: "guest"
+    description: Name of the guess cluster
+  - name: scheduler-replicas
+    default: "1"
+    description: Number of scheduler replicas
+  - name: controller-manager-replicas
+    default: "1"
+    description: Number of controller manager replicas
```

Comment:
These replicas for KCM and scheduler are not configurable as they run as daemonsets.

Reply:
These have been removed.